tempest/tvaultconf.py

```diff
@@ -3,7 +3,7 @@
 
 #If you want to cleanup all test resources like vms, volumes, workloads then set
 # following cleanup parameter value to True otherwise False
-cleanup = True
+cleanup = False
```

Comment:
Always checkin cleanup=True by default. You can change it False locally for your testing.